src/renderer/helpers/poolHelper.test.ts

```diff
@@ -359,4 +360,56 @@ describe('helpers/poolHelper/', () => {
       expect(result).toBeFalsy()
     })
   })
+
+  describe('disablePoolActions', () => {
+    const haltedChains = [ETHChain, BNBChain]
+    it('true if trading is halted for this chain', () => {
+      const result = disablePoolActions({
+        chain: BNBChain,
+        haltedChains,
```

Comment:
Maybe just move `haltedChain: [ETHChain, BNBChain]` to this line, since this is the only place where it is used